spec/spec_helper.rb

```diff
@@ -47,9 +43,6 @@
   end
 
   config.before(:each) do
-    SidekiqUniqueJobs.configure do |c|
-      c.redis_test_mode = :mock
-    end
```

Comment:
The `redis_test_mode` option does not exist in the latest version of `sidekiq-unique-jobs`.